sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/ServiceBusEndToEndTests.cs

```diff
@@ -144,131 +91,116 @@ public async Task CustomMessageProcessorTest()
                 // in addition to verifying that our custom processor was called, we're also
                 // verifying here that extensions can log
                 IEnumerable<LogMessage> messages = loggerProvider.GetAllLogMessages().Where(m => m.Category == CustomMessagingProvider.CustomMessagingCategory);
-                Assert.Equal(4, messages.Count(p => p.FormattedMessage.Contains("Custom processor Begin called!")));
-                Assert.Equal(4, messages.Count(p => p.FormattedMessage.Contains("Custom processor End called!")));
+                Assert.AreEqual(4, messages.Count(p => p.FormattedMessage.Contains("Custom processor Begin called!")));
+                Assert.AreEqual(4, messages.Count(p => p.FormattedMessage.Contains("Custom processor End called!")));
             }
         }
 
-        [Fact(Skip = "Will enable after migrating to NUnit and integrating with TestEnvironment")]
+        [Test]
         public async Task MultipleAccountTest()
         {
-            using (IHost host = new HostBuilder()
-               .ConfigureDefaultTestHost<ServiceBusTestJobs>(b =>
-               {
-                   b.AddServiceBus();
-               }, nameResolver: _nameResolver)
-               .ConfigureServices(services =>
-               {
-                   services.AddSingleton<MessagingProvider, CustomMessagingProvider>();
-               })
-               .ConfigureServices(s =>
-               {
-                   s.Configure<HostOptions>(opts => opts.ShutdownTimeout = HostShutdownTimeout);
-               })
-               .Build())
+            var (jobHost, host) = BuildHost<ServiceBusTestJobs>(host =>
+                host.ConfigureServices(services =>
+                {
+                    services.AddSingleton<MessagingProvider, CustomMessagingProvider>();
+                }));
+            using (jobHost)
             {
-                await WriteQueueMessage(_secondaryConnectionString, FirstQueueName, "Test");
+                await WriteQueueMessage(
+                    "Test",
+                    connectionString: ServiceBusTestEnvironment.Instance.ServiceBusSecondaryNamespaceConnectionString,
```

Comment:
Why use a new namespace rather than writing to a new queue in the same namespace?

Reply:
The SB function extension tests require it.